content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -42,38 +42,51 @@ Complete the DataCamp courses [Statistical Thinking in Python Part 1](https://ww
+6. Calculate the mean and median CO2 emissions for 1990. Save each answer in the variables `mean_co2_emmisions_1990` and `median_co2_emmisions_1990` respectively. Why do you think these values differ? 
+7. Calculate the minimum, maximum and interquartile range of the CO2 emissions for 1990. Please store these as variables named: `minimum_co2_emmisions_1990`, `maximum_co2_emmisions_1990` and `iqr_co2_emissions_1990` respectively. Using this information, as well as the mean and median calculated previously for this year, explain what this tells us about the distribution of CO2 emissions?
+8. Create a histogram of the CO2 emissions for 1990. Is this the distribution you expected based on your answers from questions 6 and 7? 
+9. Calculate the standard deviation and standard error of the mean for CO2 emissions in 1990. Save each answer in the variables `std_co2_emmisions_1990` and `stderr_co2_emmisions_1990` respectively. How is the standard error different from the standard deviation? 
+10. Create a line graph to show CO2 emissions in Brazil, Russia, China, India, the USA and South Africa over time. What does the graph tell you about the difference and change in C02 emissions in these countries?
+11. What is the mean and standard deviation for land area covered by forest in 1990? Save each answer in the variables `mean_land_area_covered_forest_1990` and `std_land_area_covered_forest_1990` respectively. Why do you think the standard deviation is so large? 
+12. Create histograms for land area covered by forest and percentage of area protected in 1990. Describe the distributions.
+13. Create a scatterplot with a regression line using `seaborn.regplot` to show the relationship between the proportion of land area covered by forest and the percentage of area protected in 2000.
 
   - What is the relationship between these two variables?
   - Describe any patterns in the scatterplot.
   - Do you notice any unusual/extreme values that do not fit the general trend? If you see any unusual values, briefly describe them (Who are they? In what way are they different?).
 
-11. Since neither forested land area nor protected area is normally distributed, we will need to log transform these variables in order to calculate a correlation coefficient. Log transform the variables and show the transformed distributions in a histogram.
+14. Since neither forested land area nor protected area is normally distributed, we will need to log transform these variables in order to calculate a correlation coefficient. Log transform the variables and show the transformed distributions in a histogram.
```

Comment:
Let's get them to create a dataframe of their final log transformed variables here. There are common pitfalls folk make with 0 entries so it's a good place to check.